src/Types/TransactionMetadata.purs

```diff
@@ -64,3 +69,12 @@ derive instance Generic TransactionMetadatum _
 
 instance Show TransactionMetadatum where
   show x = genericShow x
+
+instance EncodeAeson TransactionMetadatum where
+  encodeAeson' = case _ of
+    MetadataMap m -> encodeAeson' $ encodeMap m
```

Comment:
Why not `encodeTagged' "Map"`?

Reply:
I forgot to add the tag apparently, thanks.